EejitCMS.API/Models/Account.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EejitCMS.API.Models
+{
+	public class Account
+	{
+		public int Id { get; set; }
```

Comment:
consider using strings for ids

makes the number immutable, + if we run multiple backend servers for load balancing it doesn't mess stuff up if we use sequential identifiers
https://softwareengineering.stackexchange.com/a/361405

Reply:
Agree with @ocanty on making them immutable but keep them as ints.